File: Giovanna-e-Gabriela/CPP/Lista-1/ex-01.cpp
/*
Data: 30/04/2026
Autores: Gabriela Onaga e Giovanna Costa
Descrição:Ler um número inteiro. Verificar e exibir se o número é primo.
*/

#include <iostream>
using namespace std;

int main() {
    int num;
    bool primo = true;

    cout << "Digite um numero: ";
    cin >> num;

    if (num < 2) {
        primo = false;
    } else {
        for (int i = 2; i < num; i++) {
            if (num % i == 0) {
                primo = false;
            }
        }
    }

    if (primo) {
        cout << "É primo";
    } else {
        cout << "Nao é primo";
    }

    return 0;
}

File: Giovanna-e-Gabriela/Python/python-conversion/python/ex01.py
"""
    Autores: Gabriela Onaga e Giovanna Costa
    Data: 06/08/2026
    Descrição: Lê um número inteiro e informa se ele é positivo, negativo ou zero.
"""

num = int(input("Digite um número inteiro: "))

if num > 0:
    print("O número é positivo.")
elif num < 0:
    print("O número é negativo.")
else:
    print("O número é zero.")
